openstudio-extension.gemspec

```diff
@@ -31,12 +31,12 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.5.0'
 
   spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-workflow', '~> 1.3.4'
+  spec.add_dependency 'openstudio-workflow', '~> 2.0.0'
```

Comment:
I'm not sure this will work with the prerelease one `2.0.0-pre1` @nllong

Reply:
i'm going to release 2.0.0 formally now.